hiring-developers-with-readthedocs.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: October 30, 2018
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips to Hire Developers with Read the Docs
+==========================================
+
+Read the Docs is probably not the first place you think of if you are recruiting.
+However, over **7 million unique developers use Read the Docs each month** from all over the world.
+We didn't set out to build a better job board,
+but after a number of advertisers used our `ethical ads`_ for recruiting,
+we discovered that Read the Docs was a great place to find developer talent.
+
+.. _ethical ads: https://docs.readthedocs.io/en/latest/advertising/ethical-advertising.html
+
+
+Hiring with Read the Docs works 
+-------------------------------
+
+Developers are not always actively job seeking
+by browsing job boards or company careers pages.
+However, they are on Read the Docs reading about the libraries and frameworks they use.
+Even when people aren't actively looking for a new job,
+many are open to exploring new opportunities.
+
+On Read the Docs, we found that developers engage with hiring ads
+at a higher rate than any other kind of advertising.
+With hiring ads, developers get ads that are relevant and interesting
+and companies reach the folks actively using the tech they are recruiting for.
+
+.. figure:: img/2018-ctr-adtype.png
+   :alt: Click-through rate by ad type Q3 2018
+   :width: 100%
+
+
+Be specific and find the right developers
+-----------------------------------------
+
+If you are looking for Python developers,
+you can configure your hiring ad to only show on Python related documentation.
+You can further refine your audience to Read the Docs visitors in specific regions.
+Whether you are looking for a remote Python data scientist or a blockchain developer in Switzerland,
+be as specific as possible.
```

Comment:
As far as I know, we don't have too many filters to "be as specific as possible".

Reply:
Well, it was more in line with be specific about the job description. You are right that we don't have fantastic keyword targeting for jobs yet. However, that's something that can be solved over time.